app/components/AmountInput/AmountInput.tsx

```diff
@@ -0,0 +1,43 @@
+import { IMaskInput } from 'react-imask';
+
+import { cnTw } from '@/common/utils';
+
+type Props = {
+  value: string;
+  placeholder?: string;
+  isValid: boolean;
+  wrapperClassName?: string;
+  inputClassName?: string;
+  onChange: (value: string) => void;
+};
+
+export const AmountInput = ({
+  value,
+  placeholder = '0.00',
+  isValid,
+  wrapperClassName,
+  inputClassName,
+  onChange,
+}: Props) => {
+  return (
+    <div className={cnTw('px-1', wrapperClassName)}>
```

Comment:
Is this wrapper necessary? You can reduce `wrapperClassName` and `inputClassName` to just `className` if you move that wrapper outsize